packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -141,19 +141,14 @@ describe('ReactDOMInput', () => {
               value="lion"
               onChange={e => this.change(e.target.value)}
             />
-            <input
-              type="checkbox"
-              ref={n => (this.b = n)}
-              checked={true}
-              onChange={() => {}}
-            />
+            <input type="checkbox" ref={n => (this.b = n)} checked={true} />
```

Comment:
This is undoing a recently merged change.

Reply:
damn, you're right. I'll solve it, can you continue review the other files?
I had conflict only with this file and the ReactFiber of reconciler package